### PR TITLE
Added support for screen mode devices

### PR DESCRIPTION
**Pull request by dfgHiatus**

### CODE REVIEW

File: Tobii/Device.cs [hunk 1]
```diff
@@ -44,16 +47,51 @@ public Device(ILogger logger, nint api, string deviceUrl, string license = "")
...
+        if (res != tobii_error_t.TOBII_ERROR_NO_ERROR || _device == nint.Zero)
+        {
+            throw new Exception("Failed to get device info: " + res);
+        }
+
+        // This will either be HMD or Peripheral
+        return info.integration_type == "HMD";
+    }
+
+    private void GetCapabilities(ILogger logger, nint _device)
```
Comment (ghostiam) [inline Tobii/Device.cs:76]: Unused code?

File: Tobii/Tobii.StreamEngine.Interop.cs [hunk 2]
```diff
@@ -618,6 +618,11 @@ public static tobii_error_t tobii_get_face_type(IntPtr device, out string face_t
             return result;
         }
 
+        internal static tobii_error_t tobii_gaze_data_subscribe(nint device)
```
Comment (ghostiam) [inline Tobii/Tobii.StreamEngine.Interop.cs:621]: Unused code?

File: VRCFT-Tobii-Advanced.csproj [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
```
Comment (ghostiam) [inline VRCFT-Tobii-Advanced.csproj:1]: This file contains unnecessary changes, please roll them back.

File: VRCFT-Tobii-Advanced.sln [hunk 4]
```diff
@@ -1,16 +1,25 @@
 ﻿
```
Comment (ghostiam) [inline VRCFT-Tobii-Advanced.sln:1]: This file contains unnecessary changes, please roll them back.

**Comment by ghostiam** [comment]
Thanks for the PR.

Add to the README which devices are supported, otherwise only Crystal is mentioned there now.

File: Tobii/Device.cs (hunk 1 above)
Comment (dfgHiatus) [reply Tobii/Device.cs]: Resolved in ca3bda1e

File: Tobii/Tobii.StreamEngine.Interop.cs (hunk 2 above)
Comment (dfgHiatus) [reply Tobii/Tobii.StreamEngine.Interop.cs]: Resolved in ca3bda1e

File: VRCFT-Tobii-Advanced.csproj (hunk 3 above)
Comment (dfgHiatus) [reply VRCFT-Tobii-Advanced.csproj:1]: VRCFT targets `net7.0-windows10.0.19041.0, x64`. I can remove the local reference to `VRCFaceTracking.Core.dll`, but should we keep this to ensure module compatibility? As with the Babble module, etc.

File: VRCFT-Tobii-Advanced.sln (hunk 4 above)
Comment (dfgHiatus) [reply VRCFT-Tobii-Advanced.sln:1]: This issue is tied to the .`csproj` modifications. I will update this once that issue is addressed

File: VRCFT-Tobii-Advanced.csproj (hunk 3 above)
Comment (ghostiam) [reply VRCFT-Tobii-Advanced.csproj:1]: Unfortunately, I can’t even make a build with the target: `net7.0-windows10.0.19041.0`, at the same time, everything is fine with `net7.0` and works with VRCFT.
On each computer the path to the file will be different, so I made a Libs folder where you need to copy the `VRCFaceTracking.Core.dll` file(I forgot to mention this in the readme).

Comment (ghostiam) [reply VRCFT-Tobii-Advanced.csproj:1]: I was able to figure out why the build wasn't working. Everything is fine now.

But we need to do something about the path to the library...
We can replace `..\..\..\..\..\Program Files` with `C:\Program Files`, but this will not work for everyone and when VRCFT is updated the build will break again, because the path contains the VRCFT version.

But at the moment, I think the `Libs` folder is the best solution.
I saw this solution in mods for VRC.

Comment (dfgHiatus) [reply VRCFT-Tobii-Advanced.csproj:1]: To confirm, just drop a copy of `VRCFaceTracking.core.dll` into our Libs?

File: VRCFT-Tobii-Advanced.csproj [hunk 5]
```diff
@@ -37,4 +31,10 @@
         <CopyToOutputDirectory>Always</CopyToOutputDirectory>
       </AdditionalFiles>
     </ItemGroup>
+
+    <ItemGroup>
+      <Reference Include="VRCFaceTracking.Core">
+        <HintPath>Libs\VRCFaceTracking.Core.dll</HintPath>
+      </Reference>
```
Comment (ghostiam) [inline VRCFT-Tobii-Advanced.csproj:38]: Please return: `<Private>False</Private>`, this will force not to copy this library when building, since it is not needed for the module to work.

Comment (dfgHiatus) [reply VRCFT-Tobii-Advanced.csproj:38]: With this kind of syntax? Sorry:

```
<Reference Include="VRCFaceTracking.Core">
        <Private>False</Private>
        <HintPath>Libs\VRCFaceTracking.Core.dll</HintPath>
</Reference>
```

Comment (ghostiam) [reply VRCFT-Tobii-Advanced.csproj:38]: Yes, that's right, that's what I did.

Comment (dfgHiatus) [reply VRCFT-Tobii-Advanced.csproj:39]: Added in [805e634](https://github.com/ghostiam/VRCFT-Tobii-Advanced/pull/1/commits/805e6347af9e4479d856fe0feb1a015c763c7a93)
